src/components/EnvironmentSelect.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Dropdown, DropdownProps } from 'semantic-ui-react';
+import { Environment } from './types';
+import 'semantic-ui-css/semantic.css';
+import { css } from 'emotion';
+
+export interface EnvironmentSelectProps {
+    env: Environment;
+    disabled: boolean;
+
+    onEnvironmentChange(type: Environment): void;
+}
+
+const dropdownEnvironments = Object.values(Environment).map((type: string) => ({
+    text: type,
+    value: type,
+}));
+
+export const EnvironmentSelect: React.FC<EnvironmentSelectProps> = ({
+    env,
+    disabled,
+    onEnvironmentChange,
+}) => {
+    const handleEnvironmentChange = (
+        event: React.SyntheticEvent<HTMLElement>,
+        data: DropdownProps
+    ) => {
+        onEnvironmentChange(data.value as Environment);
+    };
+
+    return (
+        <div className={dropdownContainerCls}>
+            <span>Please Select Environment</span>
+
+            <Dropdown
```

Comment:
Instead of dropdown (that required 2 clicks) maybe we could you something like https://react.semantic-ui.com/elements/button/#group-variations-group-floated which only require 1 click